src/services/image.service.ts

```diff
@@ -42,8 +42,31 @@ export class ImageService {
     }).save();
   }
 
-  public async readRecipeImages(id: string) {
-    return Image.find({ recipeId: id });
+  public async readRecipeImages(recipeId: string) {
+    return Image.find({ recipeId });
+  }
+  public async readRecipeImage(imageId: string): Promise<Buffer | null> {
+    const image = await Image.findById(imageId);
```

Comment:
Check if `const { image } = ...` will work or not

Reply:
@MoodupDev
Property 'image' does not exist on type 'ImageDocument | null'
for
`const { image } = await Image.findById(imageId);`